src/fl_draw.cxx

```diff
@@ -592,67 +592,21 @@ void fl_draw_check(Fl_Rect bb, Fl_Color col) {
 } // fl_draw_check()
 
 /**
-  Draw a potentially small, filled circle as a GUI element.
+  Draw a potentially small, filled circle using a given color.
 
```

Comment:
As Matthias mentioned, the new function would draw circles of any size, hence this should read:
```
Draw a filled circle using a given color.
```

Reply:
The "potentially small" expression is Albrecht's when introducing function `fl_draw_circle()` in 1.4. I think it makes sense because this function does draw all circle sizes but exists only because small circles may need special processing.

Reply:
OK, at that time when I introduced it, that was something I wanted to emphasize. But anyway, I'm fine with leaving it.